nominal/core/__init__.py

```diff
@@ -37,4 +38,5 @@
     "CheckViolation",
     "DataReviewBuilder",
     "WriteStream",
+    "MultiProcessedWriteStream",
```

Comment:
I think this is "beta" enough that we shouldn't be exporting it just yet.

This would actually be a good candidate for exposing an `experimental` or `beta` subpackage.